server/controllers/cars/getCarAdminByName.js

```diff
@@ -0,0 +1,17 @@
+const { getCarByName } = require('../../database/queries');
+
+const getCarAdminByName = async (req, res, next) => {
+  try {
+    const { carName } = req.query;
```

Comment:
```suggestion
    const { name } = req.query;
```
That would be better